src/pages/Authorization/Authorization.css

```diff
@@ -2,96 +2,99 @@ body {
     background-color: #F8F8F8;
 }
 
-.form_auth_block {
+.auth-block {
+    background: #F8F8F8;
+    display: block;
     width: 520px;
     height: 717px;
     margin: auto;
     margin-top: 50px;
-    background: #F8F8F8;
-}
-
-.form_auth_block_content{
-    display: block;
 }
 
-.form_auth_block_head_text {
+.auth-block-head-text {
     color: #0A0A0C;
+    margin-top: 30px;
+    margin-bottom: 0;
     text-align: center;
     font-size: 32px;
     font-family: Outfit, sans-serif;
     font-weight: 600;
     letter-spacing: 0.32px;
-    margin-top: 30px;
-    margin-bottom: 0;
 }
 
-.form_auth_block_text {
+.auth-block-info-text {
     color: #414143;
-    text-align: center;
-    font-size: 16px;
-    font-family: Outfit, sans-serif;
     margin-top: 10px;
     margin-bottom: 10px;
+    text-align: center;
+    font-family: Outfit, sans-serif;
+    font-size: 16px;
 }
 
-.form_auth_block label {
-    display: block;
+*, *::before, *::after {
```

Comment:
@v3r3tnxv, что это?

Reply:
The CSS code *, *::before, *::after is a selector that targets all elements on a web page, as well as their ::before and ::after pseudo-elements. The * selector is a universal selector that matches any element, while ::before and ::after are pseudo-elements that allow you to insert content before or after an element's content.
Using this selector in CSS allows you to apply styles to all elements on a page, as well as their ::before and ::after pseudo-elements. For example, you could use this selector to set the box-sizing property to border-box for all elements, which can help simplify layout and sizing calculations.
It's worth noting that using a universal selector like * can have performance implications, as it can cause the browser to apply styles to a large number of elements. It's generally recommended to use more specific selectors whenever possible to minimize the number of elements affected by a given style rule.